docs/ci/ci-remove-dco-check.md

```diff
@@ -0,0 +1,11 @@
+# Remove DCO Sign-off Check
+
+## Overview
+
+Presently, all repositories under our `instructlab` GitHub org require contributors to add a [Developer Certificate Origin (DCO)](https://developercertificate.org/) sign-off in each commit to indicate that they agree to the DCO legal terms. While these DCO sign-offs can be a great way for us to validate that a contributor agrees to the DCO legal terms, many upstream projects actively use the DCO without explicitly requiring contributors to add "Signed-off-by" on their commits. So in other words, we can still require contributors to agree to the DCO terms, but we don't necessarily _need_ contributors to add DCO sign-offs in each commit.
```

Comment:
```
many upstream projects actively use the DCO without explicitly requiring contributors to add "Signed-off-by" on their commits.
```
Concrete example might be helpful

Reply:
Yep, will update that.

Here's an example from the `gnuradio/volk` repository:
https://github.com/gnuradio/volk/blob/308948abf8384bb4bf6467e14b585df708789782/docs/CONTRIBUTING.md?plain=1#L22

and the DCO they're referencing is located here:
https://github.com/gnuradio/volk/blob/main/docs/DCO.txt

Updating now!

Reply:
Updated. Let me know what you think